template.tpl

```diff
@@ -1512,7 +1560,7 @@ const libraryName = data.instanceName ? data.instanceName + '.' : '';
 // Build and normalize initialization options map
 const manualOptions = normalizeTable(data.initManualOptions, 'key', 'value') || {};
 
-const initOptions = (data.initOptions === 'manual' ? manualOptions : data.initOptions) || {};
+const initOptions = Object.assign({'mp_loader': 'gtm'}, (data.initOptions === 'manual' ? manualOptions : data.initOptions) || {});
```

Comment:
Object.assign is fine with ignoring falsey values, so you could simplify a little bit
```suggestion
const initOptions = Object.assign({'mp_loader': 'gtm'}, data.initOptions === 'manual' ? manualOptions : data.initOptions);
```